pkg/kudoctl/packages/types.go

```diff
@@ -8,6 +8,21 @@ const (
 	APIVersion = "kudo.dev/v1beta1"
 )
 
+var (
+	// Implicits is a set of usable implicits
+	// defined in render.go
+	Implicits = map[string]bool{
```

Comment:
this has only one usage in verify, we might as well move it there and make it not public